flixel/math/FlxMath.hx

```diff
@@ -568,4 +568,13 @@ class FlxMath
 		return (a > 0) ? a : -a;
 	}
 	
+	/**
+	 * Floors the argument. Same as Math.floor, but faster.
+	 * WARNING: This will not work for NaN or numbers that are outside the bounds of a 32-bit integer.
+	 */
+	public static inline function fastFloor(f:Float):Int
+	{
+		return f >= 0 ? Std.int(f) : Std.int(f - 1);
```

Comment:
We need to carefully profile this a very simple perf test on all platforms, to ensure it's actually faster than ```Math.floor``` alternative.